tests/Extensions/DecimalExtensionsShould.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Kros.Extensions;
+using Xunit;
+
+namespace Kros.Utils.UnitTests.Extensions
+{
+    public class DecimalExtensionsShould
+    {
+        [Fact]
+        public void ReturnNumberRoundedDown()
+        {
+            decimal input = 10.124m;
+            decimal actual = input.Round(2);
+            decimal expected = 10.12m;
+            Assert.Equal(expected, actual);
```

Comment:
We don't use standard assertion system. Use [Fluent Assertion](https://fluentassertions.com/introduction) instead.

Reply:
Resolved